syntax = "proto3";

import "google/protobuf/wrappers.proto";

package proto.common;

message Endpoint {
  string url = 1;
}

message KeySpaceRange {
  string database_id = 1;
  string table_id = 2;
  google.protobuf.StringValue start_key = 3; // start/end_key must be optional
  google.protobuf.StringValue end_key = 4;
}
